base/iostream.jl

```diff
@@ -16,8 +16,9 @@ mutable struct IOStream <: IO
     name::AbstractString
     mark::Int64
     lock::ReentrantLock
+    _dolock::Bool
```

Comment:
Why the underscore? Is this field somehow more private than others?

Reply:
It's just me getting increasingly paranoid :)

Reply:
It's almost as if we need a way to actually prevent people from messing with private internals 😁